src/prommis/solvent_extraction/sx_model_flowsheet_dynamic.py

```diff
@@ -0,0 +1,186 @@
+m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Ce"].fix(1e-9)
+m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Pr"].fix(1e-9)
+m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Nd"].fix(1e-9)
+m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Sm"].fix(1e-9)
+m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Gd"].fix(1e-9)
+m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Dy"].fix(1e-9)
+
+m.fs.solex.mscontactor.aqueous_inherent_reaction_extent[0, :, "Ka2"].fix(0)
+
+m.fs.solex.mscontactor.aqueous_inlet_state[:].flow_vol.fix(4.4)
+
+m.fs.solex.mscontactor.aqueous[0, :].flow_vol.fix(4.4)
+
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Al"].fix(7.54e-10)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Ca"].fix(4.955e-9)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Fe"].fix(1.491e-7)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Sc"].fix(321.34)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Y"].fix(5.67e-6)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["La"].fix(1.78e-05)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Ce"].fix(4.019e-5)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Pr"].fix(6.73e-6)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Nd"].fix(1.82e-5)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Sm"].fix(3.285e-6)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Gd"].fix(1.55e-6)
+m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Dy"].fix(9e-7)
+
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Al"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Ca"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Fe"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Sc"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Y"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["La"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Ce"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Pr"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Nd"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Sm"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Gd"].fix(1e-9)
+m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Dy"].fix(1e-9)
+
+m.fs.solex.mscontactor.organic_inlet_state[:].flow_vol.fix(62.01)
+
+m.fs.solex.mscontactor.organic[0, :].flow_vol.fix(62.01)
+
+"""
+Initialization of the model, which gives a good starting point.
+
+"""
+
+initializer = BlockTriangularizationInitializer()
+initializer.initialize(m.fs.solex)
+assert initializer.summary[m.fs.solex]["status"] == InitializationStatus.Ok
+
+"""
+Solution of the model and display of the final results.
+
+"""
+
+solver = SolverFactory("ipopt")
+solver.options["bound_push"] = 1e-8
+solver.options["mu_init"] = 1e-8
+solver.solve(m, tee=True)
```

Comment:
1. Is there any dynamic behavior in this example? As far as I can tell there is no disturbance, thus this doesn't demonstrate much.
2. Do you need to be changing `bound_push` and `mu_init`? If you need to change these, it generally indicates that the model is poorly scaled.

Reply:
2. I actually did not write this, this were given in the model when the prommis repo was being developed.
1. Can you elaborate the question? Are you asking whether there is any change with respect to time?

Reply:
Yes - is there any change with time. If there isn't, you aren't really testing much,

Reply:
There is change with time, in both the phases